js/demo/demo.js

```diff
@@ -257,6 +257,7 @@
                         "helptext" : "",
                         "conditions" : "",
                         "external_source" : "true",
+                        "jmespath" : "",
```

Comment:
Is it required to have this property? That would mean all existing templates would break. Better not add it to the demo in cases it isn't used.